src/components/Navbar/Navbar.js

```diff
@@ -5,14 +5,14 @@ import "./Navbar.scss"
 function Navbar() {
   return (
     <div className="Navbar">
-         <div className="navbar__left">
-            <div className="navbar__icon"><img alt="Logo" src="assets/skillChain.svg"/></div>
+        <div className="Navbar__left">
+            <div className="Navbar__icon"><img alt="Logo" src="./../../assets/skillChain.svg"/></div>
```

Comment:
It would be better to import logo here (on the top of the file)
e.g. 
import Logo from "./assets/skillchainLogoLight.svg";

and then here src={Logo}